helm/aws-load-balancer-controller/values.yaml

```diff
@@ -91,6 +91,8 @@ updateStrategy: {}
 # serviceAnnotations contains annotations to be added to the provisioned webhook service resource
 serviceAnnotations: {}
 
+deploymentAnnotations: {}
```

Comment:
Please add a comment for this value, something like:
```
# deploymentAnnotations contains additional annotations for the controller deployment
deploymentAnnotations: {}
```

Reply:
Added comment, test.values and REAME.md

https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/2611/commits/e2fe0424383fc7aaae08d798ce64d684ff989335